marc_to_solr/spec/lib/config_spec.rb

```diff
@@ -368,9 +368,9 @@ def fixture_alma_record(fixture_name)
   end
   describe 'excluding locations from library facet' do
     it 'when location is online' do
-      expect(@online['location_code_s']).to include 'elf1'
-      expect(@online['location_display']).to include 'Online - *ONLINE*'
-      expect(@online['location']).to be_nil
+      expect(@online['location_code_s']).to include 'online$elf1'
+      expect(@online['location_display']).to include 'Electronic Access - elf1 Internet Resources'
+      expect(@online['location']).to eq ['Electronic Access']
     end
```

Comment:
As far as I understand we shouldn't have elfs at all. Is this an attempt to pass the voyager specs?

Reply:
ok, I understand that this is only for the voyager specs. We will update it and remove the elfs when we clean the config_spec from the voyager fixtures.